django_auth0/utils.py

```diff
@@ -11,3 +12,23 @@ def get_config():
         'AUTH0_CALLBACK_URL': settings.AUTH0_CALLBACK_URL,
         'AUTH0_SUCCESS_URL': settings.AUTH0_SUCCESS_URL,
     }
+
+
+def get_token(*, a0_config: dict=None, audience=None) -> str:
```

Comment:
I think we need to have support for Python 2 and 3 at lease the next version should be the last one then we can just drop Python 2.x support. With typing annotations all users using Python 2.x will be broken.